By John Steele Scott, 22/2/2005.

Without this patch, pic30-elf-gcc complains with:
pic30-elf-gcc: installation problem, cannot exec `pic30-elf-cc1': No such file or directory

--- ./v1.10.src/XC16_v1_10_branch/gcc/gcc/config/pic30/pic30.h.original.1	2008-12-12 11:51:30.000000000 +0800
+++ ./v1.10.src/XC16_v1_10_branch/gcc/gcc/config/pic30/pic30.h	2008-12-12 11:53:46.000000000 +0800
@@ -270,15 +270,6 @@
 #undef   STARTFILE_SPEC
 #define   STARTFILE_SPEC   ""
 
-/* making STANDARD_EXEC_PREFIX and STANDARD_BINDIR_PREFIX point to the same
-   directory will cause make_relative_paths to make no change - ie look in the 
-   gcc executable's directory.  */
-#undef STANDARD_EXEC_PREFIX
-#undef STANDARD_BINDIR_PREFIX
-#define STANDARD_EXEC_PREFIX "/bin/"
-#define STANDARD_LIBEXEC_PREFIX "/bin/"
-#define STANDARD_BINDIR_PREFIX "/bin/bin"
-
 /* By default, the GCC_EXEC_PREFIX_ENV prefix is "GCC_EXEC_PREFIX", however
    in a cross compiler, another environment variable might want to be used
    to avoid conflicts with the host any host GCC_EXEC_PREFIX */
